examples/3ds/create_subscription.php

```diff
@@ -0,0 +1,129 @@
+<?php
+require_once('vendor/autoload.php');
+
+use Money\Money;
+use Univapay\UnivapayClient;
+use Univapay\Enums\Period;
+use Univapay\Enums\SubscriptionPlanType;
+use Univapay\Enums\ThreeDSMode;
+use Univapay\Enums\TokenType;
+use Univapay\Resources\Authentication\AppJWT;
+use Univapay\Resources\PaymentData\Address;
+use Univapay\Resources\PaymentData\PhoneNumber;
+use Univapay\Resources\PaymentData\TokenThreeDS;
+use Univapay\Resources\PaymentMethod\CardPayment;
+use Univapay\Resources\PaymentThreeDS;
+use Univapay\Resources\Subscription\ScheduleSettings;
+use Univapay\Resources\Subscription\SubscriptionPlan;
+
+/**
+ * Example 1: of creating a subscription with a 3DS on Recurring Token
+ */
+$storeAppToken = AppJWT::createToken('token', 'secret');
+$client = new UnivapayClient($storeAppToken);
+$paymentMethod = new CardPayment(
+    'test@test.com',
+    'PHP example',
+    '4242424242424242',
+    '02',
+    '2030',
+    '123',
+    TokenType::RECURRING(),
+    null,
+    new Address(
+        'test line 1',
+        'test line 2',
+        'tokyo',
+        'tokyo',
+        'jp',
+        '101-1111'
+    ),
+    new PhoneNumber(PhoneNumber::JP, '12910298309128'),
+    null,
+    null,
+    null,
+    new TokenThreeDS(
+        true,
+        null,
+        "https://ec-site.example.com/3ds/complete" // redirect url when 3ds transaction is completed
+    )
+);
+
+$token = $client->createToken($paymentMethod)->awaitResult();
+$token->threeDSIssuerToken(); // information issuer token for 3DS authentication
+// Complete the 3DS transaction process using the provided issuer token
+// After completing the 3DS transaction, redirection will occur based on the URL provided in the TokenThreeDS object
```

Comment:
What you have here is correct for recurring token 3DS flow, so let's move that to another example file specific for recurring token 3DS.

Instead, we might want to add the subscription MPI example instead